### ISSUE TITLE
API Connector Details page should contain frequency of connector use

### ISSUE THREAD

**Issue by honghuac** [syndesisio/syndesis#1537]
API Connector Details page should contain the **frequency of connector use** in any integration. This frequency value appears in the page prior, and should also appear in the details page.

![api-client-connector-details-petstore](https://user-images.githubusercontent.com/8625482/36132954-4b0675ca-10b5-11e8-9330-584563083d62.png)

**Comment by zregvart** [comment]
Any opinion @syndesisio/uxd I would think this would be easy to add.

**Comment by sjcox-rh** [comment]
@honghuac Could you perhaps share a screenshot of what this info looks like on the previous page? I can't find it, thanks!

In the meantime, I agree with @zregvart, should be pretty easy. Off the bat I think we can just add another field under the summary (top portion) and label it "Frequency of Use" (@TovaCohen, if you have input that'd be awesome!) 

But I may change my mind once I see the example :]

**Comment by TovaCohen** [comment]
I think Hong is referring to this display, which appears after you successfully upload a Swagger spec and Syndesis creates the API connector:

![image](https://user-images.githubusercontent.com/25067106/36155953-c1438852-10a3-11e8-85f7-14ef239c154e.png)

However, on this page, it is not clear what is meant by "Used by integrations x times" because an integration does not directly use a connector, and this is a list of connectors. Should this be: "x integrations use a connection based on this connector."?  Or should this be, "x connections created from this connector."???
Or is there a way to calculate how many times a connection that was created from this connector was actually used in the execution of an integration? 

Once that is clear and decided, then the page displayed above needs to be revised. The Connector Details page would then have one or more new fields with labels that clearly indicate the metric(s) we are providing, so it might be:

Connections Created from This Connector   [ x   ]
or 
Integrations That Use Connections Created from This Connector [ x  ]
or maybe
Connection Use [  x  ] -- this one would need a tooltip to explain what it is, if it makes sense to provide an indication of how many times connections created from this connector and added to integrations were actually used. 

Bottom line - what information would be useful to the user?

**Comment by amysueg** [comment]
Great comments @TovaCohen 

What is the user really interested in here? @chirino @kcbabo? 
1. The number of connections created from a particular connector?
2. How many integrations use connections created from a particular connector?

And I agree that the API Client Connectors list view would need to change (per your suggestions) based on the answers and then we can determine what to put on the custom connector details view.

**Comment by michael-coker** [comment]
@amysueg reviewing the UXD backlog and pinging you to see if there has been any movement or if you want to follow up on this?
